rust/blockstore/src/memory/storage.rs

```diff
@@ -195,7 +195,7 @@ impl<'referred_data> Readable<'referred_data> for &'referred_data str {
 }
 
 // TODO: remove this and make this all use a unified storage so we don't have two impls
-impl Writeable for Int32Array {
+impl Writeable for Vec<i32> {
```

Comment:
nit should we rename Int32Array to Veci32

Also should we just quickly make this u32 while in here?